apps/frontend/src/app/pages/company/applications-job/applications-job.component.html

```diff
@@ -0,0 +1,72 @@
+<div class="container mx-auto min-h-screen px-3">
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/5714ea0e-f7b0-4fda-b753-d79e2fce4ccd)

When the gets smaller it would be nice if the cards were centered